depfinder/main.py

```diff
@@ -319,7 +319,7 @@ def simple_import_search(path_to_source_code, remap=True, ignore=None):
     path_to_source_code : str
     remap : bool, optional
         Normalize the import names to be synonymous with their conda/pip names
-    ignore : str, optional
+    ignore : str or list, optional
```

Comment:
I'd rather make this a list than "str or list" and then expect the caller of `main` to update `ignore` to be a list instead of a string. cc @ocefpaf for possible breaking change for your code if you're using this as a library.

Then you can delete line 350 if we expect users to provide a list here.